src/components/player-info.js

```diff
@@ -186,10 +186,11 @@ AFRAME.registerComponent("player-info", {
       }
     }
 
+    const avatarEl = this.el.querySelector("[avatar-audio-source]");
     if (this.data.muted) {
-      APP.isAudioPaused.add(this.el);
+      APP.isAudioPaused.add(avatarEl);
     } else {
-      APP.isAudioPaused.delete(this.el);
```

Comment:
Does this need to be `avatarEl` in `remove` too?

Reply:
Isn't that the next line?